cutadapt.wdl

```diff
@@ -79,9 +79,12 @@ task Cutadapt {
         # Hence we use compression level 1 here.
         Int compressionLevel = 1  # This only affects outputs with the .gz suffix.
         Int cores = 4
-        String memory = "4G"
+        String? memory
+        Int? runtimeMinutes
         String dockerImage = "quay.io/biocontainers/cutadapt:2.8--py37h516909a_0"
     }
+    Int estimatedRuntimeMin = 1 + ceil(size([read1, read2], "G")  * 12.0 / cores)
+    String estimatedMemory = "~{300 + 100 * cores}M"
```

Comment:
I'm wondering if it might be better to set these as defaults of the inputs, then the method of estimation would also be shown by WDL-AID. (Also less logic, since we won't need to `select_first` later).